app/user/[id]/verify-location/page.js

```diff
@@ -0,0 +1,152 @@
+"use client"
+
+import { set } from "lodash"
+import Link from "next/link"
+import { useEffect, useState } from "react"
+import cn from 'classnames';
+import { useRouter } from "next/navigation";
+
+
+
+export default function VerifyLocation({ params }) {
+  const router = useRouter();
+
+  const [user, setUser] = useState([])
+  const [loc, setLoc] = useState()
+  const [loading, setLoading] = useState(false)
+
+
+  const getUser = async () => {
+    try {      
+      const response = await fetch(`/api/customer?term=${params.id}`)
+      const result = await response.json()
+      setUser(result.results[0])
+      getLocation();
+    } catch (error) {
+    }
+  }
+
+  useEffect(() => {
+    getUser()
+  }, [])
+
+  async function showPosition(position) {
+    let res = await getInWords(
+      position.coords.longitude,
+      position.coords.latitude
+    )
+    setLoc(res);
+  }
+
+  async function getInWords(longitude, latitude) {
```

Comment:
what does this function do?

Reply:
it converts the co-ordinate to human readable physical address